gammapy/irf/edisp/map.py

```diff
@@ -26,15 +26,16 @@ class EDispMap(IRFMap):
     Parameters
     ----------
     edisp_map : `~gammapy.maps.Map`
-        the input Energy Dispersion Map. Should be a Map with 2 non spatial axes.
+        The input Energy Dispersion Map. Should be a Map with 2 non spatial axes.
         migra and true energy axes should be given in this specific order.
     exposure_map : `~gammapy.maps.Map`, optional
         Associated exposure map. Needs to have a consistent map geometry.
 
     Examples
     --------
-    .. doctest::
+    ::
```

Comment:
is the code executed in doctest this way?

Reply:
I'm not sure why a doctest was included here, they are not included in other Examples sections?